docs/thirdparty.rst

```diff
@@ -37,7 +37,20 @@ Third Party Applications
 .. content list start
 
 
-.. replace this line with the first entry
+LiveStreamLinkGUI
+---------------------
```

Comment:
The lines of section headers have to match the length of the headline text:
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections

It still renders fine, though, which wonders me a bit... 😕